app/pages/SummaryPage.scala

```diff
@@ -623,6 +623,8 @@ case class SummaryPage(cyaDiversionService: CYADiversionService) {
     }
     (claimPIA, incomeSectionFinishedPage, sectionFinished) match {
       case (Some(true), Some(false), None)       => TaskListTag.CanNotStart
+      case (Some(true), Some(false), Some(true)) => TaskListTag.CanNotStart
+      case (Some(true), Some(false), Some(false)) => TaskListTag.CanNotStart
```

Comment:
Could this be simplified to:
```suggestion
      case (Some(true), Some(false), _) => TaskListTag.CanNotStart
```